doc/api/inspector.md

```diff
@@ -0,0 +1,111 @@
+# Inspector
+
+> Stability: 1 - Experimental
+
+The `inspector` module provides an API for interacting with the V8 inspector.
+
+It can be accessed using:
+
+```js
+const inspector = require('inspector');
+```
+
+## Class: inspector.Session
+
+The `inspector.Session` class is a primary means of dispatching messages to
+the V8 inspector back-end and receiving message responses and notifications.
+
+### Constructor: new inspector.Session()
+<!-- YAML
+added: REPLACEME
+-->
+
+Create a new instance of the `inspector.Session` class. The resulted inspector
+session needs to be connected through [`session.connect()`][] before
+messages can be sent to the inspector backend.
+
+`inspector.Session` is an [`EventEmitter`][] with the following events:
+
+### Event: 'inspectorNotification'
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object} The notification message object
+
+Emitted when any notification from the V8 Inspector is received.
+
+```js
+session.on('inspectorNotification', (message) => console.log(message.method));
+// Debugger.paused
+// Debugger.resumed
+```
+
+It is also possible to subscribe only to notifications with specific method:
+
+### Event: &lt;inspector-protocol-method&gt;
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object} The notification message object
+
+Emitted when an inspector notification is received that has method field set
+to the `<inspector-protocol-method>` value.
+
+The following snippet installs a listener on the [`Debugger.paused`][]
+event, and prints the reason for program suspension whenever program
+execution is suspended (through breakpoints, for example):
+
+```js
+session.on('Debugger.paused', ({params}) => console.log(params.hitBreakpoints));
+// [ '/node/test/inspector/test-bindings.js:11:0' ]
+```
+
+### session.connect()
+<!-- YAML
+added: REPLACEME
+-->
+
+Connects a session to the inspector back-end. An exception will be thrown
+if there is already a connected session esteblished either through API or by
+a front-end connected to the Inspector WebSocket port.
+
+### session.post(method[, params][, callback])
+<!-- YAML
+added: REPLACEME
+-->
+
+* method {string}
+* params {Object}
+* callback {Function}
+
+Posts a message to the inspector back-end. Optional `callback` will be notified
```

Comment:
I'd drop the word "Optional."  The function prototype makes it clear it's optional and "Optional callback" reads awkward (ad "The optional callback" isn't really an improvement, IMO.)